docs/README.md

```diff
@@ -67,6 +67,25 @@ $ export IONOSCLOUD_PASSWORD='password'
 $ rspec spec
 ```
 
+### Getting input from a file
+
+In order to make passing JSON or array arguments easier, we allow you to specify a file which holds the arguments for a command. This is done with the
+`--extra-config EXTRA_CONFIG_FILE` or `-e EXTRA_CONFIG_FILE`. The file is expected to contain a JSON which should map the name of arguments to their expected values. Here is an example of such a file.
+
+```json
+{
+  "ionoscloud_username": "username",
+  "ionoscloud_password": "password",
+  "name": "test_datacenter_name",
+  "location": "de/txl"
+}
+```
+
+When using the command `knife ionoscloud datacenter create --extra-config EXTRA_CONFIG_FILE` where EXTRA_CONFIG_FILE is the name of the file containing the JSON above, the arguments described in the file will be used, however if any or them are set in any other way, the values from the file will be ignored. Running the command `knife ionoscloud datacenter create --location us/las --extra-config EXTRA_CONFIG_FILE` will create a datacenter in the 'us/las' location. In the same way if the values for ionoscloud_username and ionoscloud_password are already set in the knife.rb file, the contents from the JSON will be ignored.
```

Comment:
For me, it is a little bit unclear: `where EXTRA_CONFIG_FILE is the name of the file` -- the name of the file represents only the file name or the absolute path of the file? Should the file be present in the same path where the command is ran? Maybe adding some details would help the user.